earthquakes.py

```diff
@@ -29,14 +31,64 @@ def get_data():
     # See the README file for more information.
     ...
 
+    with open('text.json', 'w') as output_file:
+        output_file.write(text)
+
+    
     # We need to interpret the text to get values that we can work with.
     # What format is the text in? How can we load the values?
-    return ...
+    return  json.loads(text)
+
 
 def count_earthquakes(data):
     """Get the total number of earthquakes in the response."""
     return ...
 
+def get_year(earthquake):
+    """Extract the year in which an earthquake happened."""
+    timestamp = earthquake['properties']['time']
+    # The time is given in a strange-looking but commonly-used format.
+    # To understand it, we can look at the documentation of the source data:
+    # https://earthquake.usgs.gov/data/comcat/index.php#time
+    # Fortunately, Python provides a way of interpreting this timestamp:
+    # (Question for discussion: Why do we divide by 1000?)
+    year = date.fromtimestamp(timestamp/1000).year
+    return year
+
+
+def plot_number_per_year(earthquakes):
+    years = []
+    for earthquake in data["features"]:
+        if earthquake['properties']['type'] == 'earthquake':
+            years.append(get_year(earthquake))
+
+    counts = {}
+    #unique_years = []
+
+    for year in years:
+        if year not in counts.keys():
+            counts[year] = 1
+
+        else:
+            counts[year] += 1
+    X = []
+    Y = []
+    for year,count in counts.items():
+        X.append(year)
+        Y.append(count)
```

Comment:
This is correct but it could be expressed more concisely.
You already have the numbers you want in the `counts` dictionary. You can get them into lists in a simpler way:
```suggestion
    X = list(counts.keys())
    Y = list(counts.values())
```
Alternatively, there is a shorthand for your approach of "unzipping" the dictionary entries: 
```suggestion
    X, Y = zip(*counts.items())
```